source/reference/minio-mc/mc-batch-generate.rst

```diff
@@ -0,0 +1,308 @@
+.. _minio-mc-batch-generate:
+
+=====================
+``mc batch generate``
+=====================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc batch
+
+.. mc:: mc batch generate
+
+.. versionchanged:: MinIO RELEASE.2022-10-08T20-11-00Z or later
+
+Syntax
+------
+
+.. start-mc-batch-generate-desc
+
+The :mc:`mc batch generate` command creates a basic YAML file for the specified job type.
+
+.. end-mc-batch-generate-desc
+
+After the MinIO client creates the YAML file, open it in your preferred text editor tool to customize the job description for your needs.
+A job definition may have as many defined job tasks as needed.
+There is no predefined limit to the number of job tasks any one YAML file can contain.
+
+For descriptions of the types of jobs you can define in a batch and details of the YAML format for each type, see :ref:`job types <minio-batch-job-types>`.
```

Comment:
Simplify:

See :ref:`job types` for the supported jobs you can generate

Reply:
Thanks for braining my words.